salt/template.py

```diff
@@ -221,6 +221,12 @@ def check_render_pipe_str(pipestr, renderers, blacklist, whitelist):
     try:
         if parts[0] == pipestr and pipestr in OLD_STYLE_RENDERERS:
             parts = OLD_STYLE_RENDERERS[pipestr].split('|')
+        if len(parts) > 1 and 'py' in parts[1:]:
```

Comment:
Slices don't raise an IndexError, you can simplify this to just
```suggestion
        if 'py' in parts[1:]:
```